src/test/java/org/jitsi/meet/test/util/MeetUtils.java

```diff
@@ -224,8 +224,9 @@ public Boolean apply(WebDriver driver)
         }
         catch(Throwable error)
```

Comment:
I don't get why Throwable is caught here given that there is specific timeout exception type. It could be interesting if some other error is thrown here - it could be logged at least.